### PR TITLE
Move eks-operator to 1.0.9-rc1

### PR DESCRIPTION
**Pull request by thedadams**

### CODE REVIEW

**Review by cmurphy — APPROVED** [review]

**Review by nickgerace — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -88,7 +88,7 @@ require (
 	github.com/prometheus/common v0.10.0
 	github.com/rancher/apiserver v0.0.0-20210209001659-a17289640582
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
-	github.com/rancher/eks-operator v1.0.9
+	github.com/rancher/eks-operator v1.0.9-rc1
```
Comment (dramich) [inline go.mod:91]: Is this going backwards in versions?

Comment (thedadams) [reply go.mod:91]: @dramich We mistakenly tagged v1.0.9 in eks-operator. I would like to delete that tag and will do so if this PR is merged.

Comment (dramich) [reply go.mod:91]: It's been out for 2 weeks, that's way too long to drop a tag

**Comment by thedadams** [comment]
Closing because we are going with different versioning.

**Pull request closed without merging**